sql-aggregates/on-the-map.sql

```diff
@@ -0,0 +1,4 @@
+select count(*) as "totalCities"
+from "cities"
+join "countries" using ("countryId")
+where "countries"."countryId" = '103'
```

Comment:
You should do this for all countries, not just country 103. And select the county name as well.